src/pip/_internal/cli/progress_bars.py

```diff
@@ -47,7 +50,13 @@ def _rich_progress_bar(
             TimeRemainingColumn(),
         )
 
-    progress = Progress(*columns, refresh_per_second=30)
+    progress = Progress(
+        *columns,
+        refresh_per_second=30,
+        console=Console(
+            force_terminal=bar_type == "forced",
```

Comment:
It should not initialize new Console object because of missing global Console settings, like in https://github.com/pypa/pip/pull/11048
Also, your changes covers another pr https://github.com/pypa/pip/pull/11024

Reply:
By "covers" do you just mean we will run into merge conflicts?

And I can see now that making a new console instance there could be an issue -- how do you recommend I change this option in the global console settings?

Reply:
> By "covers" do you just mean we will run into merge conflicts?

I think, using `auto` as a default value and `on` for forcing colors is better idea
>  how do you recommend I change this option in the global console settings?

I think you should also just use `reconfigure(force_terminal=...)`

Reply:
I see. I will try using reconfigure.

As for it clashing with your PR, I will keep this one separate for now. If necessary I could branch off of your branch (or just redo this change once yours is merged) and just add this change there for "on" mode.